README.md

```diff
@@ -3,42 +3,49 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 ![ci tests](https://github.com/opendp/opendp/actions/workflows/smoke-test.yml/badge.svg)
 
-[OpenDP](https://opendp.org) is a community effort to build trustworthy, open-source software tools for statistical analysis of sensitive private data. This repository contains OpenDP Core, a library of differential privacy algorithms powering OpenDP.
+OpenDP is a modular library of statistical algorithms that adhere to the definition of [differential privacy](https://en.wikipedia.org/wiki/Differential_privacy). It can be used to build applications of privacy-preserving computations, using a number of different models of privacy. OpenDP is implemented in Rust, with bindings for easy use from Python.
 
-## *WARNING*
-Please note that this library is still under development and not ready for production use. An important element of the OpenDP project is a formal vetting process that all library components must undergo, to verify their privacy characteristics. The code in this repository hasn't yet undergone that vetting. In addition to privacy concerns, it also likely has programming bugs.
+The architecture of OpenDP is based on a conceptual framework for expressing privacy-aware computations. This framework is described in the paper,
+[A Programming Framework for OpenDP](https://projects.iq.harvard.edu/files/opendp/files/opendp_programming_framework_11may2020_1_01.pdf).
 
-Feel free to explore the code, but *please do not yet rely on it for any privacy-sensitive applications*.
+OpenDP (the library) is part of the larger [OpenDP Project](https://opendp.org), a community effort to build trustworthy, open-source software tools for analysis of private data.
 
-## Overview
-OpenDP Core is a library of differential privacy algorithms for performing statistical analysis of private data. It consists of two conceptual layers:
+## Status
 
-* A theoretical framework for expressing privacy-aware operations. This framework is described in the paper,
-[A Programming Framework for OpenDP](https://projects.iq.harvard.edu/files/opendp/files/opendp_programming_framework_11may2020_1_01.pdf).
+OpenDP is under development, and we aim to [release new versions](https://github.com/opendp/opendp/releases) frequently. It's a work in progress, but we feel that OpenDP already can be used to build some useful applications. We also hope that it may be a vehicle for exploring new ideas in privacy. We welcome you to try it, and look forward to feedback on the usability of the library! However, please be aware of the following limitations.
```

Comment:
```suggestion
OpenDP is under development, and we aim to [release new versions](https://github.com/opendp/opendp/releases) frequently. It's a work in progress, but we feel that OpenDP already can be used to build some useful applications. We also hope that it may be a vehicle for exploring new ideas in privacy. We welcome you to try it and look forward to feedback on the usability of the library! However, please be aware of the following limitations.
```

Reply:
Seeing "work in progress" here reminds me that we have a badge at the top of the repo that says "Project Status: WIP". I'm glad these are in sync.